core/src/main/java/org/lflang/federated/launcher/FedLauncherGenerator.java

```diff
@@ -349,20 +349,40 @@ private String getRtiCommand(List<FederateInstance> federates, boolean isRemote)
     return String.join("\n", commands);
   }
 
+  private String rtiRequiredVersion = "0.2.x";
```

Comment:
This string defines the RTI version required by lfc. Where should this be defined? THe TargetConfig, or maybe FileConfig?

Reply:
I think version matching needs to be a little more sophisticated. Several versions of the RTI could be compatible with several versions of federates.  See https://github.com/lf-lang/reactor-c/pull/521#pullrequestreview-2749699933 for a suggestion about how to handle this.